src/LibYear/Output/Json/DoubleFormatter.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace LibYear.Output.Json;
+
+internal sealed class DoubleFormatter : JsonConverter<double>
```

Comment:
Let's add some tests for this bad boy as well

Reply:
Could we also name it `DoubleConverter` to match the convention of the base class and `DateTimeConverter`?